src/containers/execution-environment/registry-list.tsx

```diff
@@ -447,6 +447,22 @@ class ExecutionEnvironmentRegistryList extends React.Component<
     );
   }
 
+  private updateRegistries() {
+    ExecutionEnvironmentRegistryAPI.list(this.state.params).then((result) => {
+      const isAllCompleted = result.data.data.every(
+        (task) => task.last_sync_task.state === 'completed',
```

Comment:
```suggestion
        (task) => ['canceled', 'completed', 'failed'].includes(task.last_sync_task.state),
```

(at least that's what task deail does)